src/components/pages/RoleSignup/Styles/signup.css

```diff
@@ -0,0 +1,24 @@
+.row {
+    display: flex;
+    flex-direction: column;
+}
+
+.col1, .col2 {
+    min-width: 100%;
+    display: flex;
+    justify-content: center;
+    margin-top: 2%;
+}
+
+.card1, .card2 {
+    display: flex;
+}
+
+.title {
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.divider {
+    margin: 4em;
+}
```

Comment:
There should be a new line at the end of each component block. I hope Ryan sees that I'm being nit-picky. LOL. Wink